internal/painter/gl/gl_es.go

```diff
@@ -1,5 +1,6 @@
-// +build gles arm arm64,!darwin
+// +build gles arm arm64
```

Comment:
the previous change in this file was good.
it is compiled on M1 with previous changes.